docs/tutorials/quickstart_cogshop_run.ipynb

```diff
@@ -293,13 +293,13 @@
     "    shop_case_write = ShopCaseWrite(\n",
     "        externalId=EXAMPLE_CASE_EXTERNAL_ID,\n",
     "        # The time range SHOP is optimized over\n",
-    "        startTime=datetime.datetime(2023, 9, 14, 22),\n",
-    "        endTime=datetime.datetime(2023, 9, 24, 22),\n",
+    "        startTime=datetime.datetime(2024, 4, 31, 22),\n",
+    "        endTime=datetime.datetime(2023, 5, 2, 22),\n",
```

Comment:
```suggestion
    "        startTime=datetime.datetime(2024, 5, 31, 22),\n",
    "        endTime=datetime.datetime(2023, 6, 2, 22),\n",
```
To match the new example file (Stavanger)

Reply:
Noticed here that the year was wrong too -- I will update